dcm-anon/utils/tags.py

```diff
@@ -0,0 +1,378 @@
+# https://northstar-www.dartmouth.edu/doc/idl/html_6.2/DICOM_Attributes.html
```

Comment:
Just a suggestion, Since you wanted these tags to be extendable why not put it in a dataclass as an enum and provide an interface to add much more Enums into the class?

Reply:
or a normal list in a class with extendable methods

Reply:
After discussion decided that we would use inbuilt pydicom for tags and have a class with methods
To print tags
To print tag name 
To append tags
To append by tag name
And delete